packages/tradershub/src/features/cfd/screens/CFDSuccess/CFDSuccess.tsx

```diff
@@ -23,10 +26,10 @@ type TProps = {
 );
 
 type PlatformDetails = {
```

Comment:
since all of keys are optional you can remove optional sign and use `Partial<PlatformDetails>` instead